client/Models.js

```diff
@@ -43,6 +43,7 @@ export class Operation {
         this.date        = has(arg, 'date') && new Date(arg.date);
         this.amount      = has(arg, 'amount') && arg.amount;
         this.binary      = (maybeHas(arg, 'binary') && arg.binary) || null;
+        this.appDetails  = (maybeHas(arg, 'appDetails') && arg.appDetails) || null;
```

Comment:
I didn't abandon the plan but I have no dedicated time to do this for the moment.

Moreover, it's really difficult because we have to change every konnector using this doctype.